### PR TITLE
created reviews table

### PR DESCRIPTION
**Pull request by ethanysc**

### CODE REVIEW

File: db/migrate/20180724152308_create_reviews.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+class CreateReviews < ActiveRecord::Migration[5.2]
+  def change
+    create_table :reviews do |t|
+      t.integer :overall_rating, null: false
+      t.text :body
+      t.integer :rides_rating
```
Comment (dpickett) [inline db/migrate/20180724152308_create_reviews.rb]: if we validate for inclusion in the model, we should have these be `null: false` and / or set a reasonable default

Comment (ethanysc) [reply db/migrate/20180724152308_create_reviews.rb]: We want all the ratings except overall_rating to be optional. Should I use `numericality` instead? We do not want set `null: false` or a default constraint in the database-level.

File: app/models/review.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+class Review < ApplicationRecord
+  validates :overall_rating, presence: true, numericality: { greater_than_or_equal_to: 0, less_than_or_equal_to: 5 }
```
Comment (dpickett) [inline app/models/review.rb]: I would probably put each validation on its own line for improved readability:

```
validates :overall_rating,
  presence: true,
  numericality: { greater_than_or_equal_to: 0, less_than_or_equal_to: 5 }

File: db/migrate/20180724152308_create_reviews.rb (hunk 1 above)
Comment (dpickett) [reply db/migrate/20180724152308_create_reviews.rb]: ok 👍

File: spec/models/review_spec.rb [hunk 3]
```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe Review, type: :model do
```
Comment (dpickett) [inline spec/models/review_spec.rb]: well done.

**Pull request merged**